packages/client/src/Tests/mappingHelpers/findAsset.test.js

```diff
@@ -0,0 +1,49 @@
+/* global it expect */
+
+import findAsset from '../../Components/Mapping/helpers/findAsset'
+
+const allForces = [
```

Comment:
So this `allForces` looks like it might be significantly different, so it might be worth making an alternate `allForces` file for this, or if the differences in data don't HAVE to be there, then just alter your test to test for the generic `allForces` code from above.